middleware/traffic_enrich/main.go

```diff
@@ -28,24 +31,37 @@ func main() {
 		hex.Decode(buf, encoded)
 
 		t := time.Now()
-		process(buf)
-		DDClient.Histogram("traffic_replay.latency", float64(time.Since(t))/1e9, []string{"type:total"}, 1)
+		process(buf, s3Loader)
+		DDClient.Histogram(
+			"traffic_replay.latency",
+			float64(time.Since(t))/1e9,
+			[]string{"type:total"},
+			1,
+		)
 	}
 }
 
-func process(buf []byte) {
+func process(buf []byte, s3Loader *s3Loader) {
 	// First byte indicate payload type, possible values:
 	//  1 - Request
 	//  2 - Response
 	//  3 - ReplayedResponse
 	payloadType := buf[0]
 	headerSize := bytes.IndexByte(buf, '\n') + 1
-	// header := buf[:headerSize-1]
+	header := buf[:headerSize-1]
 
 	// Header contains space separated values of: request type, request id, and request start time (or round-trip time for responses)
-	// meta := bytes.Split(header, []byte(" "))
+	meta := bytes.Split(header, []byte(" "))
 	// For each request you should receive 3 payloads (request, response, replayed response) with same request id
 	// reqID := string(meta[1])
+
+	// time.Time.UnixNano()
```

Comment:
what are all the commented out lines of code for?

Reply:
copied from the example, and then found we do not need them. Let me clean it up.